### PR TITLE
chore(users): Fixes Friendable methods broken by removal of deprecated functions

### PR DESCRIPTION
**Pull request by mrclay**
Have not tested these.

### CODE REVIEW

**Comment by mrclay** [comment]
Ugh, more stuff in Friendable was broken.

**Comment by mrclay** [comment]
Alternately we could get rid of Friendable altogether.

File: engine/classes/ElggUser.php [hunk 1]
```diff
@@ -590,7 +590,13 @@ public function getFriendsObjects($options = array(), $limit = 10, $offset = 0)
...
 	public function countObjects($subtype = "") {
 		elgg_deprecated_notice("\ElggUser::countObjects() is deprecated. Use elgg_get_entities()", 1.9);
-		return count_user_objects($this->getGUID(), $subtype);
+		$options = [
+			'count' => true,
+		];
+		if ($subtype) {
+			$options['subtype'] = $subtype;
+		}
+		return (int)$this->getObjects($options);
```
Comment (jdalsem) [inline engine/classes/ElggUser.php]: should this also not be replaced with a get entities?

Comment (mrclay) [inline engine/classes/ElggUser.php]: These are rarely used. I wanted to fix in the simplest way since we don't have any tests.

**Comment by ewinslow** [comment]
Too late to remove `Friendable` now. Can't introduce more breaking changes other than the few already slated for 2.0.

Pulling this in.

**Pull request merged**